src/aof.c

```diff
@@ -1355,14 +1378,18 @@ int loadAppendOnlyFiles(aofManifest *am) {
         }
     }
 
-    /* Load BASE AOF. */
+    serverAssert(am->base_aof_info != NULL ||
+        listLength(am->incr_aof_list) != 0);
```

Comment:
nit pick. i prefer just one line, it's not long enough to justify ugly line break IMHO

Reply:
modified